src/store/document-drive.ts

```diff
@@ -187,56 +184,39 @@ export function useFileNodeDocument(
                 status: 'LOADING',
             });
 
-            // if the selected file node didn't change then does nothing
-            if (!changed) {
-                return;
```

Comment:
The strandUpdate handler was being removed and not added again when the current document was changed.